src/rubicon/objc/eventloop.py

```diff
@@ -33,10 +33,19 @@
     # with an underscore; that was reverted for RC1. See
     # https://github.com/python/cpython/issues/127949 and
     # https://github.com/python/cpython/issues/134657 for details.
-    from asyncio import (  # noqa: I001
-        AbstractEventLoopPolicy,
-        DefaultEventLoopPolicy,
-    )
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
```

Comment:
perhaps mostly just curious....but why not define this in `pyproject.toml`?


[edit] my guess is to take advantage of auto code removal during linting when this isn't important anymore.